frontend/src/components/latest-post-card.tsx

```diff
@@ -1,5 +1,5 @@
 import { useNavigate } from 'react-router-dom';
-import linkIcon from '@/assets/svg/link.svg';
+// import linkIcon from '@/assets/svg/link.svg';
 import Post from '@/types/post-type';
```

Comment:
Kindly remove the comments, never have code comments in the PR

Reply:
Will remove the comments

Reply:
You have not yet removed @Ashu0216